Gemfile

```diff
@@ -0,0 +1,8 @@
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+# Specify your gem's dependencies in shimmer.gemspec
+gemspec
+gem 'chrome_remote', github: 'andrewhao/chrome_remote', branch: 'event-tweaks'
```

Comment:
I've forked the wire protocol wrapper at https://github.com/andrewhao/chrome_remote/pull/1. We have to link to it from the Gemfile, but obviously when this gem goes out this will go back into the gemspec.

We should either try to open a PR back to the host project or publish our own.